task/control-plane-agent/src/mgs_common.rs

```diff
@@ -620,17 +616,30 @@ impl MgsCommon {
             version
         }));
 
-        match self.sprot.versioned_rot_boot_info(version)? {
-            SpVersionedRotBootInfo::V1(v1) => {
-                Ok(GwRotBootInfo::V1(MgsRotState::from(v1).0))
+        // Force update of this code if new variants are introduced
+        static_assertions::const_assert_eq!(
+            GwRotBootInfo::HIGHEST_KNOWN_VERSION,
+            3u8
```

Comment:
Can you put the constant `3` somewhere to give a better idea what it's tracking (`sprot-api` perhaps?)

Reply:
I don't think it should go in `sprot-api`, since it's related to the SP-MGS communication; maybe just declare it as
```rust
const EXPECTED_HIGHEST_KNOWN_MGS_VERSION: u8 = 3u8;
```